ci/jenkins/test.sh

```diff
@@ -195,7 +195,7 @@ function deliver_antrea_windows {
     export_govc_env_var
 
     cp -f build/yamls/*.yml $WORKDIR
-    docker save -o antrea-ubuntu.tar projects.registry.vmware.com/antrea/antrea-ubuntu:latest
+    docker save -o antrea-ubuntu.tar "${DOCKER_REGISTRY}/antrea/antrea-ubuntu:latest"
```

Comment:
do we need to do a `docker tag` after loading the image on the Nodes? The image name may not match the Antrea yaml manifest.

Reply:
Thanks for reminding. Then I think the original code is good enough... Changed the code back